### ISSUE TITLE
Validation required is not trigger on textarea when form submit

### ISSUE THREAD

**Issue by crossRT** [huei90/angular-validation#227]
Hi huei90, thanks for your efforts that contribute to this useful validation in the angular word, it's really helpful!
But now I have a problem: text area is not validated when form submit. 
Here's my textarea in `jade`:
````
textarea.form-control(ng-model="inputs.address" placeholder="Address" validator="required" valid-method="blur")
````

And here's my button to submit:
````
button.btn.btn-primary(validation-submit="form" ng-click="submit()")
````

All other fields are validated, error msg has shown properly. But only text area got no error msg if I empty it. Once I fill in all other fields except the text area, the form will still not submit because of the text area.
Please do let me know if I miss out something.
thanks ya =)

**Comment by huei90** [comment]
1. do you have `form(name="Form")` given the property name
2. what's your validator require condition
3. can you please paste the rest of the code (all other fields) so that I can debug

**Comment by crossRT** [comment]
Hi huei90,
1. Yes, I have the name for the name. And I did pass it to `validation-submit`
2. I'm using the default validator rules `required` and validate-method `blur`
3. yes, please refer to below.
This is my form
```jade
form.form-horizontal(name="form" novalidate)
	.form-group
	    .col-sm-2.control-label
	        label First Name
	    .col-sm-10
	        input.form-control(type="text" name="firstName" ng-model="inputs.firstName" placeholder="First Name" validator="required" valid-method="blur")

	.form-group
	    .col-sm-2.control-label
	        label Address
	    .col-sm-10
	        textarea.form-control(ng-model="inputs.address" placeholder="Address" validator="required" valid-method="blur")

    button.btn.btn-primary(ng-disabled="loading" validation-submit="form" ng-click="submit()")
        i.fa.fa-save.fa-fw
        | Save
```

To support bootstrap, I do some config to $validationProvider:
```coffeescript
exports.app.config ["$validationProvider", ($validationProvider)->
    $validationProvider.showSuccessMessage = false;

    angular.extend $validationProvider,
        validCallback: (element) ->
            $(element).parents('.form-group:first').removeClass('has-error');
        invalidCallback: (element) ->
            $(element).parents('.form-group:first').addClass('has-error');

]
```

**Comment by crossRT** [comment]
After I replied, I just found that this issue is caused by the `name` attribute for text area is missing.
It's fixed now. 
Thanks for your help.

**Comment by huei90** [comment]
Try adding a name attribute in textarea

```
textarea.form-control(name="address" ng-model="inputs.address" placeholder="Address" validator="required" valid-method="blur")
```

**Comment by huei90** [comment]
Great! Open the issue if you have others.
